h2o-py/tests/testdir_algos/automl/binomial/pyunit_automl_prostate.py

```diff
@@ -20,7 +20,7 @@ def prostate_automl():
     test = fr[2]
 
 #    aml = H2OAutoML(max_runtime_secs = 30, stopping_rounds=3, stopping_tolerance=0.001, project_name='prostate')
-    aml = H2OAutoML(max_runtime_secs = 300, stopping_rounds=2, stopping_tolerance=0.05, project_name='prostate')
+    aml = H2OAutoML(max_runtime_secs = 30, stopping_rounds=2, stopping_tolerance=0.05, project_name='prostate', exclude_algos=["GLM", "DeepLearning"])
```

Comment:
I think changing the test time is fine. Can't recall if it was set to 300 seconds for a specific reason.